plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -458,6 +458,19 @@ func ClusterRoles() []rbac.ClusterRole {
 				eventsRule(),
 			},
 		},
+		{
+			// a role for csi external components {csi-provisioner,csi-attacher,driver-registrar}
+			ObjectMeta: metav1.ObjectMeta{Name: "system:csi-external-components"},
+			Rules: []rbac.PolicyRule{
+				rbac.NewRule("create", "delete", "get", "list", "watch", "update").Groups(legacyGroup).Resources("persistentvolumes").RuleOrDie(),
+				rbac.NewRule("get", "list", "watch", "update").Groups(legacyGroup).Resources("persistentvolumeclaims").RuleOrDie(),
+				rbac.NewRule("get", "list", "watch", "create", "update", "patch").Groups(legacyGroup).Resources("events").RuleOrDie(),
+				rbac.NewRule("get", "list").Groups(legacyGroup).Resources("secrets").RuleOrDie(),
```

Comment:
I wouldn't expect listing of secrets, and I'd expect secret permissions to be granted specific to an installation in combination with the storage class setup, not globally to all secrets across all namespaces